<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainLists</class>
 <widget class="QDialog" name="MainLists">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Measurements</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/png/lists.png</normaloff>:/png/lists.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" rowspan="3">
        <widget class="QTableView" name="tvTypesMeasurement">
         <property name="autoScroll">
          <bool>false</bool>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>18</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>18</number>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pbAddTypeMeasurement">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pbDelTypeMeasurement">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Type of measurement:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0" rowspan="3">
        <widget class="QTableView" name="tvTMParams">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>18</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>18</number>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pbAddTMParam">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pbDelTMParam">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Parameters om measurement (rows of table)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0" rowspan="4">
        <widget class="QTableView" name="tvCateg">
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>18</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>18</number>
         </attribute>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pbDel_c">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Categories of measurement (columns of table)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pbAdd_c">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tvTypesMeasurement</tabstop>
  <tabstop>pbAddTypeMeasurement</tabstop>
  <tabstop>pbDelTypeMeasurement</tabstop>
  <tabstop>tvTMParams</tabstop>
  <tabstop>pbAddTMParam</tabstop>
  <tabstop>pbDelTMParam</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
